### PR TITLE
Next version of app

### PR DESCRIPTION
**Pull request by yte0**

### CODE REVIEW

**Comment by arnfinn** [comment]
FYI, I made this one: https://github.com/SKDE-Felles/qmongr-base-r/pull/3

**Comment by arnfinn** [comment]
I get this error
```
> run_app()

Listening on http://127.0.0.1:3127
Warning: Error in : `by` can't contain join column `SykehusId` which is missing from LHS
  87: <Anonymous>
Error : `by` can't contain join column `SykehusId` which is missing from LHS
```
so I guess `SykehusId` is no longer used in the latest version of `qmongrdata`?

File: inst/qmongr.yml [hunk 1]
```diff
@@ -14,7 +14,14 @@ data:
     column:
         year: "Aar"
         qi_id: "KvalIndID"
-        unit: "ShNavn"
-        unit_id: "OrgNrShus"
+        unit_name:
+            sh: "SykehusNavn"
+            hf: "HF"
+            rhf: "RHF"
+            hfshort: "Hfkortnavn"
```
Comment (arnfinn) [inline inst/qmongr.yml]: Is it possible to replace `hf: "HF"` with `hf: "Hfkortnavn"` and remove `hfshort: "Hfkortnavn"`?

File: R/mod_quality_overview.R [hunk 2]
```diff
@@ -86,69 +86,119 @@ mod_quality_overview_ui <- function(id) {
...
-   })
+  register_data <- qmongr::load_data()
+  grouped_by_HF <- qmongr::group_data(
+    register_data,
+    by = config$data$column$unit_name$hf
+  ) %>% 
+    dplyr::left_join(
+      register_data[["hospital_name_structure"]] %>% 
+          dplyr::select( 
+            .data[[config$data$column$unit_name$hfshort]],
```
Comment (arnfinn) [inline R/mod_quality_overview.R]: `hfshort` -> `hf` and change `config` instead?

File: R/mod_quality_overview.R [hunk 3]
```diff
@@ -86,69 +86,119 @@ mod_quality_overview_ui <- function(id) {
...
+        dplyr::filter(
+          .data[[config$data$column$unit_name$rhf]] %in% selected_units$RHF,
+          .data[["count"]] > 5,
+          .data[[config$data$column$year]] == input$pick_year)
+    }
 
+    if(!rlang::is_empty(selected_units$HF)) {
+      selected_data$HF <- grouped_by_HF %>% 
+        dplyr::filter(
+          .data[[config$data$column$unit_name$hfshort]] %in% selected_units$HF,
```
Comment (arnfinn) [inline R/mod_quality_overview.R]: `hfshort` -> `hf` and change `config` instead?

File: R/mod_quality_overview.R [hunk 4]
```diff
@@ -86,69 +86,119 @@ mod_quality_overview_ui <- function(id) {
...
+    selected_data$national <- national_data %>%
+      dplyr::filter(.data[["Aar"]] == input$pick_year)
+    qmongr::qi_table(selected_data, selected_units, config)
+  })
 
+  choices_treatment <- list(
+    "RHF" = grouped_by_RHF$RHF %>%
+      unique() %>%
+      sort(),
+    "HF" = grouped_by_HF[[config$data$column$unit_name$hfshort]]%>%
```
Comment (arnfinn) [inline R/mod_quality_overview.R]: `hfshort` -> `hf` and change `config` instead?

File: inst/qmongr.yml (hunk 1 above)
Comment (yte0) [reply inst/qmongr.yml]: that is a great idea!!!

**Comment by arnfinn** [comment]
How to fix this test, as an example:
```
test-config.R:28: failure: get_config
get_config() has changed from known value recorded in 'data/get_config_default.rds'.
Component "data": Component "column": Component "unit_name": Length mismatch: comparison on first 3 components
```
- Delete the file `data/get_config_default.rds`
- Rerun `devtools::test()` and get the following warning:
```
test-config.R:28: warning: get_config
Creating reference value
```
- Rerun `devtools::test()` and get no errors (for the given test).
- `commit`, `pull`, `push` etc.

**Comment by arnfinn** [comment]
Screenshot 2020-03-06 at 23 47 22

**Pull request merged**